clients/apps/web/src/components/Subscriptions/SubscriptionGroupIcon.tsx

```diff
@@ -17,26 +20,28 @@ const SubscriptionGroupIcon: React.FC<SubscriptionGroupIconProps> = ({
    * so we could set any icon dynamically.
    */
```

Comment:
This comment doesn't make sense anymore since we have a fixed enum :)

Reply:
Aye! Good catch!